README.md

```diff
@@ -260,6 +260,41 @@ We have a small collection of algorithms, one for every day of the month. Scroll
 - **Uses and Resources**
   - This question is a brain teaser and though it may not seem to have any practical application, it is used in various areas such as Crypto-Analysis and Brute-forcing cryptographic keys.
 
+### **December 14 - Ceaser Cipher**
+- **Problem**
+  - There is a specific code in which after the encoding process is done each of the characters are moved by a certain position. The number of positions by which they are moved is given by the number of characters that are present in the given word. This problem requires you to take the input string and encrypt it according to this system and add an decrypting algorithm. In the case of the last few alphabets it should move back again to the first (z->a)
+- **Example #1:**
+  ```
+  Input: and
+  Encoded output: dqg
+  ```
+- **Example #2:**
+  ```
+  Input: feel
+  Encoded output: jiip
+  ```
+
```

Comment:
Resources:
[More on the Caesar Cipher](http://practicalcryptography.com/ciphers/caesar-cipher)